<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>toolsDialog</class>
 <widget class="QDialog" name="toolsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>510</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTextEdit" name="text_input"/>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>result:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_result">
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_copy">
       <property name="text">
        <string>copy</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="rbtn_crc16">
       <property name="text">
        <string>crc16</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">rbtnGroup_calc</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="rbtn_sum">
       <property name="text">
        <string>sum</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">rbtnGroup_calc</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QRadioButton" name="rbtn_add0x33">
       <property name="text">
        <string>+0x33</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">rbtnGroup_calc</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QRadioButton" name="rbtn_ascii">
       <property name="text">
        <string>ascii</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">rbtnGroup_calc</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QRadioButton" name="rbtn_float">
       <property name="text">
        <string>float</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">rbtnGroup_calc</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QRadioButton" name="rbtn_uint32">
       <property name="text">
        <string>uint32</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">rbtnGroup_calc</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QRadioButton" name="rbtn_int32">
       <property name="text">
        <string>int32</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">rbtnGroup_calc</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="1" rowspan="2">
      <widget class="QRadioButton" name="rbtn_xorsum">
       <property name="text">
        <string>xorsum</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">rbtnGroup_calc</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="2" rowspan="2">
      <widget class="QRadioButton" name="rbtn_minus0x33">
       <property name="text">
        <string>-0x33</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">rbtnGroup_calc</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="3" rowspan="2" colspan="2">
      <widget class="QRadioButton" name="rbtn_inverse">
       <property name="text">
        <string>inverse</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">rbtnGroup_calc</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="rbtn_fcs">
       <property name="text">
        <string>fcs</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">rbtnGroup_calc</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QRadioButton" name="rbtn_doubleFloat">
       <property name="text">
        <string>double</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">rbtnGroup_calc</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QRadioButton" name="rbtn_uint16">
       <property name="text">
        <string>uint16</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">rbtnGroup_calc</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QRadioButton" name="rbtn_int16">
       <property name="text">
        <string>int16</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">rbtnGroup_calc</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btn_exit">
       <property name="text">
        <string>exit</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_formatError">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>16</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_calc">
       <property name="text">
        <string>calc</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="rbtnGroup_calc"/>
 </buttongroups>
</ui>
